### PR TITLE
added other patterns

### PR DESCRIPTION
**Pull request by ipbabble**
Added Medical Diagnosis and Retail

### CODE REVIEW

File: catalog.yaml [hunk 1]
```diff
@@ -30,6 +30,28 @@ patterns:
...
+      on a demo implementation of an automated data pipeline for chest Xray analysis 
+      previously developed by Red Hat. The original demo can be found here. It 
+      was developed for the US Department of Veteran Affairs.
+    branch: main
+    gitRepo: https://github.com/hybrid-cloud-patterns/medical-diagnosis.git
+    maintainers:
+      - name: Validated Patterns Team
+        email: team-validated-patterns@redhat.com
+    products:
+      - Red Hat Advanced Cluster Management
```
Comment (day0hero) [inline catalog.yaml:47]: we're not deploying ACM

**Review by day0hero — CHANGES_REQUESTED** [review]
@ipbabble - can you remove advanced cluster manager from the list? Once its gone its g2g.

File: catalog.yaml (hunk 1 above)
Comment (ipbabble) [reply catalog.yaml:47]: Oh ok. Fixing

**Comment by ipbabble** [comment]
@day0hero fixed.

**Review by day0hero — APPROVED** [review]
/LGTM

**Pull request merged**